### PR TITLE
feat/4 consts for seal status codes

### LINKED ISSUE #4: Add consts for seal status codes
Right now it looks like they are unexported, something to do with ffi. Consumers should be able to pull consts out and use them to check values.

### PR DESCRIPTION
**Pull request by laser**
Fixes #4

### CODE REVIEW

File: sealed_sector_health/health.go [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package sealed_sector_health
```
Comment (magik6k) [inline sealed_sector_health/health.go:1]: I don't want to sound too nitpicky, but a shorter name like `sectorhealth` would sound better IMO

File: sealed_sector_health/health.go [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+// Health represents the healthiness of a sector managed by a
+// sector builder.
+type Health int
+
+const (
+	Unknown              Health = iota
```
Comment (magik6k) [inline sealed_sector_health/health.go]: Why not just keep those in the sectorbuilder package, with a prefix - like `HealthUnknown`, `HealthOk`?

File: sealed_sector_health/health.go [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+	Ok                          // everything is fine
+	ErrorInvalidChecksum        // sector exists, but checksum is invalid
```
Comment (magik6k) [inline sealed_sector_health/health.go]: Since `Ok` is really the only valid state, it's probably fine to drop the `Error` prefix

**Review by magik6k — COMMENTED** [review]
This review is probably a bit nitpicky, the consts will be nice to have

File: sealed_sector_health/health.go (hunk 3 above)
Comment (laser) [reply sealed_sector_health/health.go]: Sounds good.

File: sealed_sector_health/health.go (hunk 1 above)
Comment (laser) [reply sealed_sector_health/health.go:1]: Understood. I use the full name, `sealed_sector_health` to ensure that the intent is clear: These health-values apply only to sealed (and not staged) sectors.

File: sealed_sector_health/health.go (hunk 2 above)
Comment (laser) [reply sealed_sector_health/health.go]: Good question. My answer is more or less the same as to your earlier question: I don't want to have to manipulate the `const` names again if (when?) we add staged sector health to this API.

**Review by dignifiedquire — APPROVED** [review]

**Pull request merged**